### DISCUSSION TITLE
Talents that reduceaffliction with statuseffects (Crusty Seaman) give medical skill

### DISCUSSION THREAD

**Discussion by violetpuppygf** [FakeFishGames/Barotrauma#14876] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Skill gain can sometimes give skills in ways that don't feel rewarding or intentional. I made a mod that reduces afflictions under a specific set of circumstances, but I coded it like Crusty Seaman. CharacterAbilityReduceAffliction does not support conditionals, so instead I used CharacterAbilityApplyStatusEffects to apply a statuseffect that does reduceaffliction, just like Crusty Seaman. This however gives you medical skill gain like any other statuseffect that reduces affliction (e.g. bandages). I tested it and sure enough in vanilla Crusty Seaman gives medical skill for reducing bleeding. 

This isn't game breaking, but it isn't exactly satisfying either to continually get free medical skill very often. I found one person mention this in the Discord but saw no bug reports. If this is low priority to change or fix, I understand, I just get bug reports every now and then for my mod that really are problems with the base game and code and figured I'd make a bug report since people report it to me. 

### Reproduction steps

1. Use console command givetalent "Crusty Seaman"
2. Use console command giveaffliction bleeding 100
3. Observe passive medical skill gain occur

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.17.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
The issue has been fixed in our private repo now, fix coming up in the next unstable build.
